### DISCUSSION TITLE
General discussion

### DISCUSSION THREAD

**Discussion by sh1zicus** [sh1zicus/dots-hyprland-old#34] [General]
uncategorized

**Comment by sh1zicus** [comment]
I have a problem. after the trial period for cursor ended, I realized that I couldn’t do anything myself :( and the free ones are completely garbage, without claude 3.5 I’m helpless
<img src="https://github.com/user-attachments/assets/6582f59e-3429-42bf-993c-6f1f570d53fe" width="200" />

**Reply by PharmaRacist** [reply]
same but i use windsurf has that 50 propmpt per accound and doesnt need verification also it do most of the stuff

**Comment by sh1zicus** [comment]
I'm tired of Arch Linux constantly breaking
I think I need to move to another one, I think it will be nixos or fedora

**Reply by sh1zicus** [reply]
> I'm tired of Arch Linux constantly breaking I think I need to move to another one, I think it will be nixos or fedora

nice. hyprland break too

**Reply by PharmaRacist** [reply]
it its and there is alot of hyprland packages issues ill move to nobara/fedora today downloaded the iso

**Reply by sh1zicus** [reply]
> it its and there is alot of hyprland packages issues ill move to nobara/fedora today downloaded the iso

I try nixos

**Reply by PharmaRacist** [reply]
i did simple porting to fedora with ai i will try it now

**Reply by sh1zicus** [reply]
> > it its and there is alot of hyprland packages issues ill move to nobara/fedora today downloaded the iso
> 
> I try nixos

https://github.com/end-4/CirnOS

**Reply by sh1zicus** [reply]
> > > it its and there is alot of hyprland packages issues ill move to nobara/fedora today downloaded the iso
> > 
> > I try nixos
> 
> https://github.com/end-4/CirnOS

now i hate nixos too

**Reply by PharmaRacist** [reply]
fedora is bloated as fuck the performance was ugly i was wrong 
is there any immutable distro built  on arch ?

**Reply by sh1zicus** [reply]
> fedora is bloated as fuck the performance was ugly i was wrong is there any immutable distro built on arch ?

I have just tried Nixos and I will say that it is very good, only I have not tried Hyprland and AGS on it yet

The main advantage of it is that it was set up 1 time (almost forever)

**Reply by sh1zicus** [reply]
CirnOS i was can't able to install, and in the chat they say that the code is garbage there

**Reply by PharmaRacist** [reply]
the nix os principle for one time declarative config is good and thier repo is huge but it need soo much hassile

**Reply by PharmaRacist** [reply]
also what do you think of end4's he has abandoned mentaining the project

**Reply by sh1zicus** [reply]
> also what do you think of end4's he has abandoned mentaining the project

if you're talking about CirnOS, then I think i can still take a couple of hundred lines here

**Reply by sh1zicus** [reply]
> also what do you think of end4's he has abandoned mentaining the project

if it's about hyprland, then I would also stop supporting it if it were possible to do everything 1 in 1 on gnome.

**Reply by sh1zicus** [reply]
maybe I'll stop supporting arch and switch to nixos if everything goes well, and maybe I'll try to move to gnome and make everything as similar as possible there.

**Reply by PharmaRacist** [reply]
i dont like gnome tbh hyprland is improving ill search for some stable branchs /repos for arch maybe ill see the steamdeck iso since its immutable and build on arch cuz im too lazy too learm my os from scratch 🐸

**Reply by nichroot** [reply]
hey @Pharmaracist bro why did you stop the repo? 😭😭 btw can you try pikaOS? I think they have a hyprland version too, btw does your fedora script work? Thanks 🙏🏻👍🏻

**Reply by PharmaRacist** [reply]
> hey @Pharmaracist bro why did you stop the repo? 😭😭 btw can you try pikaOS? I think they have a hyprland version too, btw does your fedora script work? Thanks 🙏🏻👍🏻

I archived it because every thing became counter productive and there is stuff I cannot change like the color generation for example it's so complicated for no reason i make another dots from scratch using ags matugen and rofi because not every time ags breaks i stay helpless
The dots needs a lot of work but of you want to try it's dots-minimal

**Reply by nichroot** [reply]
> > hey @Pharmaracist bro why did you stop the repo? 😭😭 btw can you try pikaOS? I think they have a hyprland version too, btw does your fedora script work? Thanks 🙏🏻👍🏻
> 
> I archived it because every thing became counter productive and there is stuff I cannot change like the color generation for example it's so complicated for no reason i make another dots from scratch using ags matugen and rofi because not every time ags breaks i stay helpless The dots needs a lot of work but of you want to try it's dots-minimal

thanks a lot man, hope you will be able to update the install instructions soon, also will that version also have the Qur'an reader and azan reminder? it was so cool to see on the other dots

**Reply by PharmaRacist** [reply]
it will support hadeeth + tafseer also quran reader with the biggest reciters

**Reply by PharmaRacist** [reply]
if u want to contribute i opened the repo for everyone but its currently lacking alot am using rofi and waybar and inhirited some design language soon will move to ags entirely

**Reply by nichroot** [reply]
I would love to contribute, but currently I don't really know much coding and basically dumb, but I'm studying step by step and inshallah will be able to help you soon! also btw do you know @AhmedSaadi0 ? I think you could also take some inspiration for the ags code from him, also i think he is currently switching to a new one called fabric which uses python, please do check out his profile if you haven't ❣️

**Reply by PharmaRacist** [reply]
i saw ahmed's config but didnt like it because of the colors or maybe its just me , i saw fabric and ignis both use python but i dont think python is good for widgets because its slower than js hope u the best brother <3

**Reply by PharmaRacist** [reply]
> I would love to contribute, but currently I don't really know much coding and basically dumb, but I'm studying step by step and inshallah will be able to help you soon! also btw do you know @AhmedSaadi0 ? I think you could also take some inspiration for the ags code from him, also i think he is currently switching to a new one called fabric which uses python, please do check out his profile if you haven't ❣️

maybe im going to change my mind ill give it a try

**Reply by nichroot** [reply]
which one? ahmed's config? please do create a fork on yours as well please, I just really like the modifications you and sh1zicus do

**Comment by sh1zicus** [comment]
I'm still going back to arch with backups before updating, I'll try to rewrite the config from end-4 to the minimum version, but with my features to make everything work
mine doesn't work.

**Comment by PharmaRacist** [comment]
fuck i accedentally deleted the styles folder in the new dotfiles fucccccccccccck

**Comment by PharmaRacist** [comment]
i think at this point i will try to return back to my trusty dotfiles hehe but maybe i will try to add github copilot since it became free , apply the new enhancements to  the ipod widgets and make new theme / colors / wallpaper sections 🤞

**Comment by PharmaRacist** [comment]
https://github.com/Axenide/Dotfiles
guys im impressed 
im not it turned out awful its lacking alot alot

**Reply by PharmaRacist** [reply]
the comeback to my dots will be legendary 💣😌  suggest a name for the dots

**Comment by PharmaRacist** [comment]
yo @sh1zicus  why didnt you take the scss for the taskmgr 
![image](https://github.com/user-attachments/assets/3277832b-1370-4d07-8ebf-dba0b10f6912)

**Reply by enessmr** [reply]
look for the scss and make th-

oh nvm if its a app

You need to make the text white or black for the unreadable part, which idk

**Comment by PharmaRacist** [comment]
yo @sh1zicus  how can i use the ollama local stuff since i added it in the extragpt array with its auth txt but still give me wrong model 
![image](https://github.com/user-attachments/assets/e541a9ac-94db-4aef-8857-ebdacc07f3f4)
![image](https://github.com/user-attachments/assets/926c9df2-724b-40fc-a959-8f860c3c600a)

**Reply by sh1zicus** [reply]
idk

**Reply by PharmaRacist** [reply]
i have done it 
![image](https://github.com/user-attachments/assets/e51aab85-7c04-4fe1-a9db-88957b0f5ba1)

**Reply by enessmr** [reply]
`<think>` is not a valid html, or if it use xml its not defined then.

the think array is empty, I think it did not think or its smth hidden

We still could give the ChatGPT or DeepSeek look to <think>

**Reply by enessmr** [reply]
i see a matugen fish, also applied on me

![image](https://github.com/user-attachments/assets/8c62600c-3a83-4d81-8ef8-1b0bd4fe84b4)

**Comment by enessmr** [comment]
Approved look, I definetely want to switch to this dots.

Could be a branch called minimalisim for space in bar (widgets look very wide in bar and it could hide smth, we just make it 5 ws then the + button expands it to useroptions total workspaces then theres a - button which minimizes it to 5, i tried it but the js complication made my brain give error and made my brain burn at the same time cuz i literally dont know gjs js with ags and i dont know js, i know to console.log but thats web js tutorial)

**Comment by enessmr** [comment]
> yo @sh1zicus  how can i use the ollama local stuff since i added it in the extragpt array with its auth txt but still give me wrong model 
> ![image](https://github.com/user-attachments/assets/e541a9ac-94db-4aef-8857-ebdacc07f3f4)
> 

@Pharmaracist i know this error. You did not provide a API key. (if you provide it can give you not found for no reason)

Also if you did provide a API key and it gives error Not found, then we need to use Curl/Python method. then,
We need to give the message you typed to curl/python, then it could work. (I literally have NO IDEA about system message thing.)

I could implement this but i dont want today. (thursday when posted)

**Reply by sh1zicus** [reply]
Yes, it has not yet been completed

**Comment by PharmaRacist** [comment]
@enessmr these isnt my dots my dots are #https://github.com/pharmaracist/HyprNoon

**Reply by PharmaRacist** [reply]
ollama's local api stuff doesnt require api key ,btw if you use the openrouter service you have to get one

**Reply by enessmr** [reply]
ollama doesnt wanna respond on me idk why
oh i didnt know openrouter

**Reply by enessmr** [reply]
can i publish my one, i git pulled it last day since you deleted it this day

**Reply by enessmr** [reply]
I have a copy of it

**Reply by IgnacioAyerra** [reply]
<img width="1254" height="1208" alt="image" src="https://github.com/user-attachments/assets/0bbadd58-0621-41a6-87e6-e42dc84f7496" />
Sorry, don't even know how to post on here, new to the ricing setups. Im getting this error on the installer and dont even know which command or what was going on in the terminal when it happened. How can I fix it??

**Reply by enessmr** [reply]
Ahh yes the classic agsv1 bug aylur made 😩😩😩
The solution is at [koeqaife's hmy archive](https://github.com/koeqaife/hyprland-material-you) Btw the issue is in there as I remember

**Reply by enessmr** [reply]
The issue is gone.

**Comment by enessmr** [comment]
Looking kinda outdated. but a great fork to merge your dots :D
